sdcm/cluster.py

```diff
@@ -4235,6 +4257,16 @@ def install_scylla_manager(self, node):
     def _scylla_install(self, node):
         node.update_repo_cache()
         node.clean_scylla()
+
+        install_mode = self.params.get('install_mode')
+        try:
+            mode = InstallMode(install_mode)
+        except Exception as ex:
+            raise ValueError('Invalid install mode: {}, err: {}'.format(str(install_mode), ex)) from ex
+
+        if mode == InstallMode.WEB:
+            node.web_install_scylla(scylla_version=self.params.get('scylla_version'))
+
```

Comment:
One question here: what will happen next after the web install in this function? Will it try to install it from the repo?

Reply:
The web installer script will prepare the right repo and install the rpms.
we need to run scylla_setup as usual after the web install in this function.